README.md

```diff
@@ -23,21 +23,20 @@ Make sure you follow the instructions carefully. You will be asked to complete a
 ---------------
 ## 💭 Reflection and Self Assessment
 
-**I learned:** (repeat as needed)
+**I learned:** how to clone a repository onto my local drive, how to commit changes, and how to push changes to the remote repository so that my professor can see my edits.
 - .
 
-**I struggled with:** (repeat as needed)
+**I struggled with:** how to clone the repository as I did not know that Github already generated the link for me and I did not push my changes correctly because I tried to do it without reading the instructions.
```

Comment:
It happens, glad you were able to figure it out. In the future read through instructions fully first then work.